### DISCUSSION TITLE
Doris扩容BE，新增BE节点无法负载均衡

### DISCUSSION THREAD

**Discussion by wangguan13** [apache/doris#25347] [A - General / Q&A]
BE日志
I1012 13:53:06.394686 131817 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: Connection refused), retrying[3]...
W1012 13:53:06.394711 131817 utils.cpp:89] fail to get master client from cache. host=127.0.0.1, port=9020, code=[THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: Connection refused)

配置情况
10.185.255.86 
BE配置:priority_networks = 10.185.255.86/24
FE配置:priority_networks = 10.185.255.86/24
10.185.255.87
BE配置:priority_networks = 10.185.255.87/24

SHOW PROC '/backends';
10.185.255.87的状态为{"lastStreamLoadTime":-1,"isQueryDisabled":false,"isLoadDisabled":false}
且数据没有负载均衡

**Comment by yagagagaga** [comment]
```sql
show backends;
```
Execute the above command and see the results.

**Comment by eric107657** [comment]
解决了吗请问
